src/Components/nuget.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
This file should be removed in favor of a src/Components/build/sources.props file. You should be able to copy over https://github.com/aspnet/AspNetCore/blob/master/src/AADIntegration/build/sources.props